bin/go-rpm-integration

```diff
@@ -226,25 +226,29 @@ fullprovides(){
 
 provides() {
 goipath="${1}"
+lockfile="${2}"
 fullprovides golang-ipath "${goipath}"
 for prov in $(\
   GOPATH="${prefix}${gopath}"      \
   golist --provided                \
-	 --package-path ${goipath} \
-         ${golistflags[$goipath]}         \
+         --package-path ${goipath} \
+         $(cat "${lockfile}")      \
```

Comment:
Please don't get additional options for the `golist` this way. Plus, they are the same as in `${golistflags[$goipath]}` so they are actually duplicated.

Reply:
That's the only way I found to relay reliably flags from goinstall to autodeps in presence of multiple subpackages and default options

The ${golistflags[$goipath]} way of doing things does not work with default flags and is a lot more inconvenient to use

Reply:
Can not disagree with that

Reply:
(BTW right now goinstall won't work if you try to install a subpackage before the main package, doing the root package first is not a huge problem for me otherwise the way to fix this use case it probably something like nukink the symlink path if it already exists as a directory before creating the symlink)